CHANGES.md

```diff
@@ -18,6 +18,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * [Metronome] Updates to fix daylight saving issues.
 
+* Fix support for big sets in the ipset mgr (COPS-5229)
```

Comment:
Can you prefix that with context, e.g. "DC/OS Net:" and finish the sentence with a full stop? Meh... I understand if you don't want t revise your PR anymore, but someone has to do it in an editorial polishing step! :)